runtime/error.c

```diff
@@ -1,12 +1,103 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
 #include "balrt.h"
+#include "third-party/libbacktrace/backtrace.h"
+
+#define INITIAL_PC_COUNT 1
+#define MAX_PC_COUNT 256
+#define SKIP_FROM_END 1 // This number counts from the last stack frame
+#define THREAD 0
+
+struct backtrace_state *state;
+
+typedef struct {
+    uint32_t npcs;
+    uint32_t szpcs;
+    PC *pcs;
+} Trace;
+
+static void onError(void *data, const char *msg, int errnum) {
+    fprintf(stdout, "Error : libbacktrace : %s\n", msg);
```

Comment:
Need a name that says what it is doing, not just when it is called. Is this being called when printing the backtrace? If so, then name it accordingly.

Reply:
This is the error_callback function which is common for backtrace_simple, backtrace_create_state, and backtrace_pcinfo. That is why I followed the same pattern used for other callback functions.

Reply:
Changed the callback funcion names.